irods/models.py

```diff
@@ -22,22 +22,19 @@ class Model(six.with_metaclass(ModelBase, object)):
 class RuleExec(Model):
     id = Column(Integer, 'RULE_EXEC_ID', 1000)
     name = Column(String, 'RULE_EXEC_NAME', 1001)
+    rei_file_path = Column(String,'RULE_EXEC_REI_FILE_PATH', 1002)
     user_name = Column(Integer, 'RULE_EXEC_USER_NAME', 1003)
+    exec_time = Column(DateTime,'RULE_EXEC_TIME',    1005)
+    last_exe_time = Column(DateTime,'RULE_EXEC_LAST_EXE_TIME', 1010)
+    frequency = Column(String,'RULE_EXEC_FREQUENCY', 1006)
 
-#('RULE_EXEC_ID', 1000)
-#('RULE_EXEC_NAME', 1001)
-#('RULE_EXEC_USER_NAME', 1003)
-#----
-#('RULE_EXEC_REI_FILE_PATH', 1002)
-#('RULE_EXEC_ADDRESS', 1004)
-#('RULE_EXEC_TIME',    1005)
-#('RULE_EXEC_FREQUENCY', 1006)
-#('RULE_EXEC_PRIORITY', 1007)
-#('RULE_EXEC_ESTIMATED_EXE_TIME', 1008)
-#('RULE_EXEC_NOTIFICATION_ADDR', 1009)
-#(DateTime,'RULE_EXEC_LAST_EXE_TIME', 1010)
-#('RULE_EXEC_STATUS', 1011)
-#('RULE_EXEC_CONTEXT', 1012)
+# - Unused, or usually absent:
+
+    # exec_status = Column(String,'RULE_EXEC_STATUS', 1011)
```

Comment:
this seems... useful.    in case a rule has failed for some reason?

Reply:
Item for discuss, as this always seems to be blank.

Reply:
Will see what appears there for a failed rule. Seems like a success (so far) just leaves it blank? @alanking ?

Reply:
yes, i think so - and i just made a bad one... and status didn't change.   so yes, let's leave it out for now.   can always add it easily later.

Reply:
`RULE_EXEC_STATUS` was used in the past to indicate whether a rule was ready to be run, currently running, failed, etc. It is no longer used due to performance and the new design for the delay server doesn't require its use (yet).

Reply:
what he said.

Reply:
Ah, never mind : ) ... haven't actually looked at that field. Also need to figure a way to include it conditionally based on server version.

Reply:
no, don't worry about it.   to the future!!!